File: Angular.CRUD.Tarea.Application/Common/Models/ArticuloModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Angular.CRUD.Tarea.Application.Common.Models
{
    public class ArticuloModel
    {
        public int id_articulo { get; set; }
        public string codigo_articulo { get; set; }
        public string marca_articulo { get; set; }
        public string modelo_articulo { get; set; }
        public string descripcion_articulo { get; set; }
        public int id_tipo_insumo { get; set; }
        public int id_categoria_articulo { get; set; }
        public string? glosa_categoria_articulo { get; set; }
        public bool vigente_categoria_articulo { get; set; }
    }
}

File: Angular.CRUD.Tarea.Client/src/app/models/articuloModel.ts
export interface ArticuloModel {
    id_articulo: number
    codigo_articulo: string
    marca_articulo: string
    modelo_articulo: string;
    descripcion_articulo: string
    id_tipo_insumo: number
    id_categoria_articulo: number
    glosa_categoria_articulo: string
    vigente_categoria_articulo: boolean
}